src/Data/Config.php

```diff
@@ -165,24 +165,39 @@ static function ( $query ) {
 	 * @return string
 	 */
 	public function graphql_wp_query_cursor_pagination_stability( string $orderby, WP_Query $wp_query ) {
-
+		// Bail early if it's not a GraphQL Request.
 		if ( true !== is_graphql_request() ) {
 			return $orderby;
 		}
 
-		global $wpdb;
+		// If pre-filter hooked, return $pre_orderby.
+		$pre_orderby = apply_filters( 'graphql_pre_wp_query_cursor_pagination_stability', null, $orderby, $wp_query );
+		if ( null !== $pre_orderby ) {
+			return $pre_orderby;
+		}
 
-		// If the cursor "graphql_cursor_compare" arg is not in the query,
-		// default to using ID DESC as the stabilizer
-		if ( ! isset( $wp_query->query['graphql_cursor_compare'] ) ) {
-			return "{$orderby}, {$wpdb->posts}.ID DESC ";
+		// Bail early if disabled by connection.
+		if ( isset( $wp_query->query_vars['graphql_apply_cursor_pagination_orderby'] )
+			&& false === $wp_query->query_vars['graphql_apply_cursor_pagination_orderby'] ) {
+			return $orderby;
 		}
 
+		// Bail early if the cursor "graphql_cursor_compare" arg is not in the query,
+		if ( ! isset( $wp_query->query_vars['graphql_cursor_compare'] ) ) {
+			return $orderby;
+		}
+
+		global $wpdb;
+
 		// Check the cursor compare order
-		$order = '>' === $wp_query->query['graphql_cursor_compare'] ? 'ASC' : 'DESC';
+		$order = '>' === $wp_query->query_vars['graphql_cursor_compare'] ? 'ASC' : 'DESC';
+
+		// Get Cursor ID key.
+		$cursor = new PostObjectCursor( $wp_query->query_vars );
```

Comment:
Do we want to filter all these cursor classes before we call them, so devs can use their own?

Reply:
No cuz I don't encourage that behavior, the preferred behaviors for them to hook into filters in the classes and alter the behavior that way. And there's a good chance that if they filter and replace one instance of the class in the `orderby` callback but not the `where` callback for the same type, that would likely result in a fatal error.

Reply:
> And there's a good chance that if they filter and replace one instance of the class in the orderby callback but not the where callback for the same type, that would likely result in a fatal error.

We'd just use the same filter in both places. But I hear the desire to lock this down.